### PR TITLE
Initial interface

### PR DESCRIPTION
**Pull request by mecha**
Initial interfaces.

### CODE REVIEW

File: src/BlockInterface.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+use Dhii\Util\String\StringableInterface;
+
+/**
+ * Something that can be a block.
+ *
+ * A block is a class that is responsible for generating output.
+ *
+ * @since [*next-version*]
+ */
+interface BlockInterface extends StringableInterface
```
Comment (XedinUnknown) [inline src/BlockInterface.php]: Shall we do the same thing as we discussed before, and give this interface a distinct method, such as `getOutput()`? Not sure if this is what we had agreed on, but if you agree with this - feel free to do that.

Comment (mecha) [reply src/BlockInterface.php]: @XedinUnknown I agree. What about `render()` instead?

Comment (XedinUnknown) [reply src/BlockInterface.php]: @mecha, `render()` sounds good.

File: test/functional/BlockInterfaceTest.php [hunk 2]
```diff
@@ -59,11 +58,31 @@ public function testCanBeCreated()
         );
     }
 
+    /**
+     * Tests whether the block renders correctly.
+     *
+     * @since [*next-version*]
+     */
+    public function testRender()
```
Comment (XedinUnknown) [inline test/functional/BlockInterfaceTest.php]: Didn't really understand what you're testing here. This is an interface ;P

Comment (mecha) [reply test/functional/BlockInterfaceTest.php]: Uhh ... haha. My bad. Brain derp 😆

**Review by XedinUnknown — APPROVED** [review]

**Comment by mecha** [comment]
@XedinUnknown The Packagist listing for this package needs to be updated after merging this PR. Any commit following the [rename commit](https://github.com/Dhii/output-block-interface/pull/1/commits/6446af143043ce5e1657d3a9e14835cb28b39b74) does not seem to be detected by Packagist - it only serves 0b6ab47 as the latest. This means that dependencies have to include the repository in their `composer.json` file to get the latest version.

**Review by XedinUnknown — APPROVED** [review]

**Pull request merged**